zeppelin-server/pom.xml

```diff
@@ -43,19 +43,19 @@
       <dependency>
         <groupId>org.scala-lang</groupId>
         <artifactId>scala-library</artifactId>
-        <version>2.10.4</version>
+        <version>2.11.8</version>
```

Comment:
Following is 'ps aux | grep zeppelin' result from my system, and it doesn't not include zeppelin-server in the classpath.

```
Lees-MacBook:zeppelin-review moon$ ps aux | grep zeppelin
moon             7630   0.0  5.8  4262896 486108 s004  S    12:48PM   0:20.84 /Library/Java/JavaVirtualMachines/jdk1.7.0_79.jdk/Contents/Home/bin/java -Dfile.encoding=UTF-8 -Dlog4j.configuration=file:///Users/moon/Projects/zeppelin-review/conf/log4j.properties -Dzeppelin.log.file=/Users/moon/Projects/zeppelin-review/logs/zeppelin-interpreter-scalding-moon-Lees-MacBook.attlocal.net.log -Xms1024m -Xmx1024m -XX:MaxPermSize=512m -cp ::/Users/moon/Projects/zeppelin-review/interpreter/scalding/*:/Users/moon/Projects/zeppelin-review/zeppelin-interpreter/target/lib/*::/Users/moon/Projects/zeppelin-review/conf:/Users/moon/Projects/zeppelin-review/conf:/Users/moon/Projects/zeppelin-review/zeppelin-interpreter/target/classes org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer 59008
moon             7623   0.0  0.0  2454888   1264 s004  S    12:48PM   0:00.02 /bin/bash /Users/moon/Projects/zeppelin-review/bin/interpreter.sh -d /Users/moon/Projects/zeppelin-review/interpreter/scalding -p 59008 -l /Users/moon/Projects/zeppelin-review/local-repo/2BPXMN9Y8
moon             7619   0.0  5.0  4311156 419968 s004  S    12:47PM   0:08.45 /Library/Java/JavaVirtualMachines/jdk1.7.0_79.jdk/Contents/Home/bin/java -Dfile.encoding=UTF-8 -Xms1024m -Xmx1024m -XX:MaxPermSize=512m -Dlog4j.configuration=file:///Users/moon/Projects/zeppelin-review/conf/log4j.properties -Dzeppelin.log.file=/Users/moon/Projects/zeppelin-review/logs/zeppelin-moon-Lees-MacBook.attlocal.net.log -cp ::/Users/moon/Projects/zeppelin-review/zeppelin-server/target/lib/*:/Users/moon/Projects/zeppelin-review/zeppelin-zengine/target/lib/*:/Users/moon/Projects/zeppelin-review/zeppelin-interpreter/target/lib/*:/Users/moon/Projects/zeppelin-review/*::/Users/moon/Projects/zeppelin-review/conf:/Users/moon/Projects/zeppelin-review/zeppelin-interpreter/target/classes:/Users/moon/Projects/zeppelin-review/zeppelin-zengine/target/classes:/Users/moon/Projects/zeppelin-review/zeppelin-server/target/classes org.apache.zeppelin.server.ZeppelinServer
```

I build this branch and started without any configuration and then started scalding interpreter by running %scalding.
Could you suggest way to reproduce the problem?